src/dev/run_check_published_api_changes.ts

```diff
@@ -224,32 +235,39 @@ async function run(
     opts.help = true;
   }
 
-  const folders = ['core/public', 'core/server', 'plugins/data/server', 'plugins/data/public'];
+  const core = ['core/public', 'core/server'];
+  const plugins = [
+    'plugins/data/server',
+    'plugins/data/public',
+    'plugins/kibana_utils/common/state_containers',
+    'plugins/kibana_utils/public/state_sync',
+  ];
+  const folders = [...core, ...plugins];
 
   if (opts.help) {
     process.stdout.write(
       dedent(chalk`
         {dim usage:} node scripts/check_published_api_changes [...options]
 
-        Checks for any changes to the Kibana Core API
```

Comment:
I removed `Core` mentioned from API Extractor, because technically we now use it for plugins also.